dev/com.ibm.ws.jdbc_fat_derby/test-applications/jdbcapp/src/jdbc/fat/derby/web/JDBCDerbyServlet.java

```diff
@@ -30,9 +34,46 @@
 import componenttest.annotation.ExpectedFFDC;
 import componenttest.app.FATServlet;
 
+@DataSourceDefinitions(value = {
+                                 @DataSourceDefinition(
+                                                       name = "java:module/env/jdbc/dsfat6",
+                                                       className = "jdbc.tran.none.driver.TranNoneDataSource",
+                                                       databaseName = "memory:ds6",
+                                                       isolationLevel = Connection.TRANSACTION_NONE,
+                                                       transactional = false,
+                                                       properties = {
+                                                                      "createDatabase=create"
+                                                       }),
+                                 @DataSourceDefinition(
+                                                       name = "java:module/env/jdbc/dsfat7",
```

Comment:
Same as in the server config, these seem to be the same.  Look through all the datasourcedefs and reduce to the minimum needed.